data/tmpl/values/content5.html

```diff
@@ -66,6 +66,7 @@ <h2 class="token-awarded hidden" aria-hidden="true">Here’s your badge!</h2>
       </footer>
     </section>
     <script src="/data/js/lib/token-manager.js"></script>
+    <script src="/data/js/lib/common.js"></script>
```

Comment:
@schalkneethling True. Common logic should be loaded first in case we add stuff to it for the other JS files.